bundles/org.openhab.binding.huesync/src/main/java/org/openhab/binding/huesync/internal/api/dto/device/HueSyncDeteiledDeviceInfoWifi.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.huesync.internal.api.dto.device;
+
+/**
+ * HDMI Sync Box Device Information DTO - Wifi connection information
+ * 
+ * @author Patrik Gfeller - Initial Contribution
+ * 
+ * @see <a href=
+ *      "https://developers.meethue.com/develop/hue-entertainment/hue-hdmi-sync-box-api/#Resource%20Table">Hue
+ *      HDMI Sync Box API</a>
+ */
+public class HueSyncDeteiledDeviceInfoWifi {
```

Comment:
Probably this should be HueSyncDetailedDeviceInfoWifi -- with 'ai' rather than 'ei' ??